lib/pipekit/field_repository.rb

```diff
@@ -1,25 +1,34 @@
 module Pipekit
   module FieldRepository
-      def get_by_key(key)
-        key = Config.field_id(parent_resource, key)
-        search_fields("key", key)
-      end
+    include Repository
```

Comment:
it's a fair point, we had a lot of debate about this, we were going to get the module to inherit but turns out modules can't inherit.

I realise I hate inheritance, but I can't see a good way of doing repositories without them, and every use of them in the wild uses modules so we ended up sticking with it